2024-08-18 16:20:08.331 [info] Experiment 'pythonCreateEnvOnPipInstall' is active
2024-08-18 16:20:08.331 [info] Experiment 'pythonRecommendTensorboardExt' is active
2024-08-18 16:20:08.331 [info] Experiment 'pythonSurveyNotification' is active
2024-08-18 16:20:08.331 [info] Experiment 'pythonTerminalEnvVarActivation' is active
2024-08-18 16:20:08.331 [info] Experiment 'pythonTestAdapter' is active
2024-08-18 16:20:08.331 [error] Reading directory to watch failed [Error: ENOENT: no such file or directory, scandir '/home/fxy/Documents/2tal/.pixi/envs'
	at Object.readdirSync (node:fs:1509:26)
	at t.readdirSync (node:electron/js2c/node_init:2:11418)
	at /home/fxy/.vscode/extensions/ms-python.python-2024.12.3-linux-x64/out/client/extension.js:2:586747
	at /home/fxy/.vscode/extensions/ms-python.python-2024.12.3-linux-x64/out/client/extension.js:2:586938
	at Array.map (<anonymous>)
	at d.initWatchers (/home/fxy/.vscode/extensions/ms-python.python-2024.12.3-linux-x64/out/client/extension.js:2:586656)
	at async d.ensureWatchersReady (/home/fxy/.vscode/extensions/ms-python.python-2024.12.3-linux-x64/out/client/extension.js:2:543067)] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'scandir',
  path: '/home/fxy/Documents/2tal/.pixi/envs'
}
2024-08-18 16:20:08.331 [error] Dir "/home/fxy/Documents/2tal/.pixi/envs" is not watchable (directory does not exist)
2024-08-18 16:20:08.331 [info] Default formatter is set to null for workspace /home/fxy/Documents/2tal
2024-08-18 16:20:08.331 [info] Python interpreter path: /bin/python3
2024-08-18 16:20:08.435 [info] > pyenv which python
2024-08-18 16:20:08.435 [info] cwd: .
2024-08-18 16:20:09.239 [info] > pixi --version
2024-08-18 16:20:09.240 [error] Error: spawn pixi ENOENT
    at Process.ChildProcess._handle.onexit (node:internal/child_process:286:19)
    at onErrorNT (node:internal/child_process:484:16)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'spawn pixi',
  path: 'pixi',
  spawnargs: [ '--version' ]
}
2024-08-18 16:20:09.240 [warning] could not find a pixi interpreter for the interpreter at /bin/python3
2024-08-18 16:20:10.519 [info] Starting Pylance language server.
2024-08-18 16:24:00.723 [info] Discover tests for workspace name: undefined - uri: /home/fxy/.config/Code/User/settings.json
